domain_scoring/domain_scoring.py

```diff
@@ -110,15 +88,15 @@ def _extract_training_data_labels(self, metapath_graph: RankingGraph) -> (List[T
         metapath_labels = []
 
         for closure in metapath_graph.transitive_closures():
-            node_vector = closure.pop(0)
+            node_vector = closure.pop(0) # pop first element
```

Comment:
I think until now we always put comments one line above

Reply:
That would have a different semantic for me. A comment above comments a block of code. A comment on the same line as code only the respective line.

Reply:
Ok, as long as everyone in the team interprets it in this way, I'm fine with it.

Reply:
We need to have a style guide anyways.